infrastructure/helm-chart/charts/components/charts/api/charts/api-communication/templates/websocket/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
             httpHeaders:
             - name: Health-Check
               value: health-check
-          initialDelaySeconds: 60
+          initialDelaySeconds: 120
```

Comment:
:+1:

Reply:
this was necessary because every pod that launches in Fargate needs to wait for a Fargate node to be created